CRITICAL - 2021-06-06 08:12:26 --> Array to string conversion
#0 C:\xampp\htdocs\planmynight\app\Views\signupPage.php(103): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 103)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('signupPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\SignUp.php(24): view('signupPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\SignUp->submit()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SignUp))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 08:12:49 --> Array to string conversion
#0 C:\xampp\htdocs\planmynight\app\Views\signupPage.php(103): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 103)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('signupPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\SignUp.php(24): view('signupPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\SignUp->submit()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SignUp))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 08:15:04 --> Array to string conversion
#0 C:\xampp\htdocs\planmynight\app\Views\signupPage.php(103): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 103)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('signupPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\SignUp.php(24): view('signupPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\SignUp->submit()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SignUp))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 08:15:17 --> Array to string conversion
#0 C:\xampp\htdocs\planmynight\app\Views\signupPage.php(103): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 103)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('signupPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\SignUp.php(24): view('signupPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\SignUp->submit()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SignUp))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 08:23:45 --> Undefined array key ""
#0 C:\xampp\htdocs\planmynight\app\Views\signupPage.php(96): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 96)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('signupPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\SignUp.php(24): view('signupPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\SignUp->submit()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SignUp))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 08:23:50 --> Undefined array key ""
#0 C:\xampp\htdocs\planmynight\app\Views\signupPage.php(96): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 96)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('signupPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\SignUp.php(24): view('signupPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\SignUp->submit()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SignUp))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 08:24:54 --> Undefined array key "name"
#0 C:\xampp\htdocs\planmynight\app\Views\signupPage.php(96): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 96)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('signupPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\SignUp.php(24): view('signupPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\SignUp->submit()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SignUp))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 08:38:41 --> syntax error, unexpected token "echo"
#0 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('signupPage', Array, true)
#2 C:\xampp\htdocs\planmynight\app\Controllers\SignUp.php(10): view('signupPage')
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\SignUp->index()
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SignUp))
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-06-06 08:47:32 --> An exception occurred while executing 'INSERT INTO user (name, surname, username, password, email, role) VALUES (?, ?, ?, ?, ?, ?)' with params ["Filip", "Andric", "filipandric", "adminnnnn", "andricfilip2000@gmail.com", 0]:

Duplicate entry 'filipandric' for key 'username_UNIQUE'
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php(182): Doctrine\DBAL\Driver\AbstractMySQLDriver->convertException('An exception oc...', Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError))
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php(159): Doctrine\DBAL\DBALException::wrapException(Object(Doctrine\DBAL\Driver\Mysqli\Driver), Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'An exception oc...')
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\Connection.php(2214): Doctrine\DBAL\DBALException::driverExceptionDuringQuery(Object(Doctrine\DBAL\Driver\Mysqli\Driver), Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'INSERT INTO use...', Array)
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\Statement.php(183): Doctrine\DBAL\Connection->handleExceptionDuringQuery(Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'INSERT INTO use...', Array, Array)
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(287): Doctrine\DBAL\Statement->execute()
#5 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(1122): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->executeInserts()
#6 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(423): Doctrine\ORM\UnitOfWork->executeInserts(Object(Doctrine\ORM\Mapping\ClassMetadata))
#7 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(378): Doctrine\ORM\UnitOfWork->commit(NULL)
#8 C:\xampp\htdocs\planmynight\app\Controllers\SignUp.php(46): Doctrine\ORM\EntityManager->flush()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\SignUp->submit()
#10 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SignUp))
#11 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2021-06-06 08:51:02 --> An exception occurred while executing 'INSERT INTO user (name, surname, username, password, email, role) VALUES (?, ?, ?, ?, ?, ?)' with params ["Jovan", "Ivkovic", "jovanivkovic", "adminadmin", "jovanivko@gmail.com", 0]:

Duplicate entry 'jovanivkovic' for key 'username_UNIQUE'
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php(182): Doctrine\DBAL\Driver\AbstractMySQLDriver->convertException('An exception oc...', Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError))
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php(159): Doctrine\DBAL\DBALException::wrapException(Object(Doctrine\DBAL\Driver\Mysqli\Driver), Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'An exception oc...')
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\Connection.php(2214): Doctrine\DBAL\DBALException::driverExceptionDuringQuery(Object(Doctrine\DBAL\Driver\Mysqli\Driver), Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'INSERT INTO use...', Array)
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\Statement.php(183): Doctrine\DBAL\Connection->handleExceptionDuringQuery(Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'INSERT INTO use...', Array, Array)
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(287): Doctrine\DBAL\Statement->execute()
#5 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(1122): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->executeInserts()
#6 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(423): Doctrine\ORM\UnitOfWork->executeInserts(Object(Doctrine\ORM\Mapping\ClassMetadata))
#7 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(378): Doctrine\ORM\UnitOfWork->commit(NULL)
#8 C:\xampp\htdocs\planmynight\app\Controllers\SignUp.php(46): Doctrine\ORM\EntityManager->flush()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\SignUp->submit()
#10 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SignUp))
#11 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2021-06-06 08:56:18 --> An exception occurred while executing 'INSERT INTO user (name, surname, username, password, email, role) VALUES (?, ?, ?, ?, ?, ?)' with params ["shdfbsj", "dhfsdjdsj", "filipandric", "asdasd", "andricfilip2000@gmail.com", 0]:

Duplicate entry 'filipandric' for key 'username_UNIQUE'
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php(182): Doctrine\DBAL\Driver\AbstractMySQLDriver->convertException('An exception oc...', Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError))
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php(159): Doctrine\DBAL\DBALException::wrapException(Object(Doctrine\DBAL\Driver\Mysqli\Driver), Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'An exception oc...')
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\Connection.php(2214): Doctrine\DBAL\DBALException::driverExceptionDuringQuery(Object(Doctrine\DBAL\Driver\Mysqli\Driver), Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'INSERT INTO use...', Array)
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\Statement.php(183): Doctrine\DBAL\Connection->handleExceptionDuringQuery(Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'INSERT INTO use...', Array, Array)
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(287): Doctrine\DBAL\Statement->execute()
#5 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(1122): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->executeInserts()
#6 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(423): Doctrine\ORM\UnitOfWork->executeInserts(Object(Doctrine\ORM\Mapping\ClassMetadata))
#7 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(378): Doctrine\ORM\UnitOfWork->commit(NULL)
#8 C:\xampp\htdocs\planmynight\app\Controllers\SignUp.php(47): Doctrine\ORM\EntityManager->flush()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\SignUp->submit()
#10 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SignUp))
#11 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2021-06-06 15:15:59 --> Undefined variable $Filip
#0 C:\xampp\htdocs\planmynight\app\Controllers\Main.php(12): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 12)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\Main->index()
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-06-06 15:18:25 --> Unsupported operand types: string + string
#0 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include()
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('mainPage', Array, true)
#3 C:\xampp\htdocs\planmynight\app\Controllers\Main.php(12): view('mainPage', Array)
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\Main->index()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-06-06 15:19:32 --> Unsupported operand types: string + string
#0 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\Main->index()
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-06-06 17:08:08 --> Unrecognized field: Place_address
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1715): Doctrine\ORM\ORMException::unrecognizedField('Place_address')
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1590): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementColumnSQL('Place_address', NULL)
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1735): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementSQL('Place_address', 'Ruzveltova 10, ...', NULL)
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1072): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionSQL(Array, NULL)
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(728): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectSQL(Array, NULL, NULL, 1, NULL, NULL)
#5 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityRepository.php(233): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->load(Array, NULL, NULL, Array, NULL, 1, NULL)
#6 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(19): Doctrine\ORM\EntityRepository->findOneBy(Array)
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-06-06 17:12:13 --> Unrecognized field: Place_address
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1715): Doctrine\ORM\ORMException::unrecognizedField('Place_address')
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1590): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementColumnSQL('Place_address', NULL)
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1735): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementSQL('Place_address', 'Ruzveltova 10, ...', NULL)
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1072): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionSQL(Array, NULL)
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(728): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectSQL(Array, NULL, NULL, 1, NULL, NULL)
#5 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityRepository.php(233): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->load(Array, NULL, NULL, Array, NULL, 1, NULL)
#6 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(19): Doctrine\ORM\EntityRepository->findOneBy(Array)
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-06-06 17:12:54 --> Unrecognized field: Place_address
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1715): Doctrine\ORM\ORMException::unrecognizedField('Place_address')
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1590): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementColumnSQL('Place_address', NULL)
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1735): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementSQL('Place_address', 'Ruzveltova 10, ...', NULL)
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1072): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionSQL(Array, NULL)
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(728): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectSQL(Array, NULL, NULL, 1, NULL, NULL)
#5 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityRepository.php(233): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->load(Array, NULL, NULL, Array, NULL, 1, NULL)
#6 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(19): Doctrine\ORM\EntityRepository->findOneBy(Array)
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-06-06 17:13:24 --> Unrecognized field: Place_address
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1715): Doctrine\ORM\ORMException::unrecognizedField('Place_address')
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1590): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementColumnSQL('Place_address', NULL)
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1735): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementSQL('Place_address', 'Ruzveltova 10, ...', NULL)
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1072): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionSQL(Array, NULL)
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(728): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectSQL(Array, NULL, NULL, 1, NULL, NULL)
#5 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityRepository.php(233): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->load(Array, NULL, NULL, Array, NULL, 1, NULL)
#6 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(19): Doctrine\ORM\EntityRepository->findOneBy(Array)
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-06-06 17:14:06 --> Unrecognized field: Place_address
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1715): Doctrine\ORM\ORMException::unrecognizedField('Place_address')
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1590): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementColumnSQL('Place_address', NULL)
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1735): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementSQL('Place_address', 'Ruzveltova 10, ...', NULL)
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1072): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionSQL(Array, NULL)
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(728): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectSQL(Array, NULL, NULL, 1, NULL, NULL)
#5 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityRepository.php(233): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->load(Array, NULL, NULL, Array, NULL, 1, NULL)
#6 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(19): Doctrine\ORM\EntityRepository->findOneBy(Array)
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-06-06 17:15:56 --> Unrecognized field: Place_address
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1715): Doctrine\ORM\ORMException::unrecognizedField('Place_address')
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1590): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementColumnSQL('Place_address', NULL)
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1735): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionStatementSQL('Place_address', 'Ruzveltova 10, ...', NULL)
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(1072): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectConditionSQL(Array, NULL)
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(894): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->getSelectSQL(Array, NULL, NULL, NULL, NULL, NULL)
#5 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityRepository.php(217): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->loadAll(Array, NULL, NULL, NULL)
#6 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(19): Doctrine\ORM\EntityRepository->findBy(Array)
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-06-06 17:18:07 --> A new entity was found through the relationship 'App\Models\Entities\Owner#iduser' that was not configured to cascade persist operations for entity: App\Models\Entities\User@0000000018175f5d0000000071b99f89. To solve this issue: Either explicitly call EntityManager#persist() on this unknown entity or configure cascade persist this association in the mapping for example @ManyToOne(..,cascade={"persist"}). If you cannot find out which entity causes the problem implement 'App\Models\Entities\User#__toString()' to get a clue.
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(3499): Doctrine\ORM\ORMInvalidArgumentException::newEntitiesFoundThroughRelationships(Array)
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(388): Doctrine\ORM\UnitOfWork->assertThatThereAreNoUnintentionallyNonPersistedAssociations()
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(378): Doctrine\ORM\UnitOfWork->commit(NULL)
#3 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(41): Doctrine\ORM\EntityManager->flush()
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-06-06 17:21:00 --> Undefined constant "App\Controllers\user"
#0 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-06-06 17:21:26 --> A new entity was found through the relationship 'App\Models\Entities\Owner#iduser' that was not configured to cascade persist operations for entity: App\Models\Entities\User@0000000044a92e6b0000000004ca74fe. To solve this issue: Either explicitly call EntityManager#persist() on this unknown entity or configure cascade persist this association in the mapping for example @ManyToOne(..,cascade={"persist"}). If you cannot find out which entity causes the problem implement 'App\Models\Entities\User#__toString()' to get a clue.
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(3499): Doctrine\ORM\ORMInvalidArgumentException::newEntitiesFoundThroughRelationships(Array)
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(388): Doctrine\ORM\UnitOfWork->assertThatThereAreNoUnintentionallyNonPersistedAssociations()
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(378): Doctrine\ORM\UnitOfWork->commit(NULL)
#3 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(44): Doctrine\ORM\EntityManager->flush()
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-06-06 17:23:51 --> A new entity was found through the relationship 'App\Models\Entities\Owner#iduser' that was not configured to cascade persist operations for entity: App\Models\Entities\User@0000000057f5c98f000000006d4754ec. To solve this issue: Either explicitly call EntityManager#persist() on this unknown entity or configure cascade persist this association in the mapping for example @ManyToOne(..,cascade={"persist"}). If you cannot find out which entity causes the problem implement 'App\Models\Entities\User#__toString()' to get a clue.
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(3499): Doctrine\ORM\ORMInvalidArgumentException::newEntitiesFoundThroughRelationships(Array)
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(388): Doctrine\ORM\UnitOfWork->assertThatThereAreNoUnintentionallyNonPersistedAssociations()
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(378): Doctrine\ORM\UnitOfWork->commit(NULL)
#3 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(44): Doctrine\ORM\EntityManager->flush()
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-06-06 17:24:19 --> A new entity was found through the relationship 'App\Models\Entities\Owner#iduser' that was not configured to cascade persist operations for entity: App\Models\Entities\User@000000002b639e9600000000009342a1. To solve this issue: Either explicitly call EntityManager#persist() on this unknown entity or configure cascade persist this association in the mapping for example @ManyToOne(..,cascade={"persist"}). If you cannot find out which entity causes the problem implement 'App\Models\Entities\User#__toString()' to get a clue.
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(3499): Doctrine\ORM\ORMInvalidArgumentException::newEntitiesFoundThroughRelationships(Array)
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(388): Doctrine\ORM\UnitOfWork->assertThatThereAreNoUnintentionallyNonPersistedAssociations()
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(378): Doctrine\ORM\UnitOfWork->commit(NULL)
#3 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(56): Doctrine\ORM\EntityManager->flush()
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-06-06 17:28:58 --> Entity of type App\Models\Entities\Owner is missing an assigned ID for field  'iduser'. The identifier generation strategy for this entity requires the ID field to be populated before EntityManager#persist() is called. If you want automatically generated identifiers instead you need to adjust the metadata mapping accordingly.
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Id\AssignedGenerator.php(50): Doctrine\ORM\ORMException::entityMissingAssignedIdForField(Object(App\Models\Entities\Owner), 'iduser')
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(983): Doctrine\ORM\Id\AssignedGenerator->generate(Object(Doctrine\ORM\EntityManager), Object(App\Models\Entities\Owner))
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(1807): Doctrine\ORM\UnitOfWork->persistNew(Object(Doctrine\ORM\Mapping\ClassMetadata), Object(App\Models\Entities\Owner))
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(1764): Doctrine\ORM\UnitOfWork->doPersist(Object(App\Models\Entities\Owner), Array)
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(629): Doctrine\ORM\UnitOfWork->persist(Object(App\Models\Entities\Owner))
#5 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(42): Doctrine\ORM\EntityManager->persist(Object(App\Models\Entities\Owner))
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2021-06-06 17:30:22 --> App\Models\Entities\Owner::setIduser(): Argument #1 ($iduser) must be of type App\Models\Entities\User, int given, called in C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php on line 37
#0 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(37): App\Models\Entities\Owner->setIduser(1)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-06-06 17:31:33 --> A new entity was found through the relationship 'App\Models\Entities\Owner#iduser' that was not configured to cascade persist operations for entity: App\Models\Entities\User@00000000776d18b9000000004bb464e0. To solve this issue: Either explicitly call EntityManager#persist() on this unknown entity or configure cascade persist this association in the mapping for example @ManyToOne(..,cascade={"persist"}). If you cannot find out which entity causes the problem implement 'App\Models\Entities\User#__toString()' to get a clue.
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(3499): Doctrine\ORM\ORMInvalidArgumentException::newEntitiesFoundThroughRelationships(Array)
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(388): Doctrine\ORM\UnitOfWork->assertThatThereAreNoUnintentionallyNonPersistedAssociations()
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(378): Doctrine\ORM\UnitOfWork->commit(NULL)
#3 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(43): Doctrine\ORM\EntityManager->flush()
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-06-06 17:33:34 --> A new entity was found through the relationship 'App\Models\Entities\Owner#iduser' that was not configured to cascade persist operations for entity: App\Models\Entities\User@000000003abddb740000000015dc5677. To solve this issue: Either explicitly call EntityManager#persist() on this unknown entity or configure cascade persist this association in the mapping for example @ManyToOne(..,cascade={"persist"}). If you cannot find out which entity causes the problem implement 'App\Models\Entities\User#__toString()' to get a clue.
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(3499): Doctrine\ORM\ORMInvalidArgumentException::newEntitiesFoundThroughRelationships(Array)
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(388): Doctrine\ORM\UnitOfWork->assertThatThereAreNoUnintentionallyNonPersistedAssociations()
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(378): Doctrine\ORM\UnitOfWork->commit(NULL)
#3 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(46): Doctrine\ORM\EntityManager->flush()
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-06-06 17:38:14 --> A new entity was found through the relationship 'App\Models\Entities\Owner#iduser' that was not configured to cascade persist operations for entity: App\Models\Entities\User@000000006b641c3600000000764fb3cf. To solve this issue: Either explicitly call EntityManager#persist() on this unknown entity or configure cascade persist this association in the mapping for example @ManyToOne(..,cascade={"persist"}). If you cannot find out which entity causes the problem implement 'App\Models\Entities\User#__toString()' to get a clue.
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(3499): Doctrine\ORM\ORMInvalidArgumentException::newEntitiesFoundThroughRelationships(Array)
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(388): Doctrine\ORM\UnitOfWork->assertThatThereAreNoUnintentionallyNonPersistedAssociations()
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(378): Doctrine\ORM\UnitOfWork->commit(NULL)
#3 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(43): Doctrine\ORM\EntityManager->flush()
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2021-06-06 17:57:00 --> An exception occurred while executing 'INSERT INTO owner (jmbg, license, address, iduser) VALUES (?, ?, ?, ?)' with params ["xxxx", "300", "Bulevar Kralja Aleksandra 191, Belgrade, Serbia", 3]:

Duplicate entry 'xxxx' for key 'jmbg_UNIQUE'
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php(182): Doctrine\DBAL\Driver\AbstractMySQLDriver->convertException('An exception oc...', Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError))
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\DBALException.php(159): Doctrine\DBAL\DBALException::wrapException(Object(Doctrine\DBAL\Driver\Mysqli\Driver), Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'An exception oc...')
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\Connection.php(2214): Doctrine\DBAL\DBALException::driverExceptionDuringQuery(Object(Doctrine\DBAL\Driver\Mysqli\Driver), Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'INSERT INTO own...', Array)
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\dbal\lib\Doctrine\DBAL\Statement.php(183): Doctrine\DBAL\Connection->handleExceptionDuringQuery(Object(Doctrine\DBAL\Driver\Mysqli\Exception\StatementError), 'INSERT INTO own...', Array, Array)
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Persisters\Entity\BasicEntityPersister.php(287): Doctrine\DBAL\Statement->execute()
#5 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(1122): Doctrine\ORM\Persisters\Entity\BasicEntityPersister->executeInserts()
#6 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\UnitOfWork.php(423): Doctrine\ORM\UnitOfWork->executeInserts(Object(Doctrine\ORM\Mapping\ClassMetadata))
#7 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(378): Doctrine\ORM\UnitOfWork->commit(NULL)
#8 C:\xampp\htdocs\planmynight\app\Controllers\RegisterPlace.php(45): Doctrine\ORM\EntityManager->flush()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\RegisterPlace->submit()
#10 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterPlace))
#11 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2021-06-06 18:09:26 --> Undefined variable $place
#0 C:\xampp\htdocs\planmynight\app\Controllers\Main.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 17)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\Main->index()
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-06-06 18:33:12 --> Undefined variable $locations
#0 C:\xampp\htdocs\planmynight\app\Views\mapViewPage.php(25): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 25)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('mapViewPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MapView.php(9): view('mapViewPage')
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MapView->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MapView))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 18:34:34 --> Class 'App\Controllers\Entities\Place' does not exist
#0 C:\xampp\htdocs\planmynight\vendor\doctrine\persistence\lib\Doctrine\Persistence\Mapping\RuntimeReflectionService.php(38): Doctrine\Persistence\Mapping\MappingException::nonExistingClass('App\\Controllers...')
#1 C:\xampp\htdocs\planmynight\vendor\doctrine\persistence\lib\Doctrine\Persistence\Mapping\AbstractClassMetadataFactory.php(326): Doctrine\Persistence\Mapping\RuntimeReflectionService->getParentClasses('App\\Controllers...')
#2 C:\xampp\htdocs\planmynight\vendor\doctrine\persistence\lib\Doctrine\Persistence\Mapping\AbstractClassMetadataFactory.php(360): Doctrine\Persistence\Mapping\AbstractClassMetadataFactory->getParentClasses('App\\Controllers...')
#3 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Mapping\ClassMetadataFactory.php(85): Doctrine\Persistence\Mapping\AbstractClassMetadataFactory->loadMetadata('App\\Controllers...')
#4 C:\xampp\htdocs\planmynight\vendor\doctrine\persistence\lib\Doctrine\Persistence\Mapping\AbstractClassMetadataFactory.php(251): Doctrine\ORM\Mapping\ClassMetadataFactory->loadMetadata('App\\Controllers...')
#5 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(293): Doctrine\Persistence\Mapping\AbstractClassMetadataFactory->getMetadataFor('App\\Controllers...')
#6 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\Repository\DefaultRepositoryFactory.php(45): Doctrine\ORM\EntityManager->getClassMetadata('App\\Controllers...')
#7 C:\xampp\htdocs\planmynight\vendor\doctrine\orm\lib\Doctrine\ORM\EntityManager.php(768): Doctrine\ORM\Repository\DefaultRepositoryFactory->getRepository(Object(Doctrine\ORM\EntityManager), 'App\\Controllers...')
#8 C:\xampp\htdocs\planmynight\app\Controllers\MapView.php(11): Doctrine\ORM\EntityManager->getRepository('App\\Controllers...')
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MapView->index()
#10 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MapView))
#11 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2021-06-06 19:18:40 --> Undefined constant "App\Controllers\user"
#0 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\Main->index()
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-06-06 19:22:40 --> Undefined property: App\Controllers\Login::$sessin
#0 C:\xampp\htdocs\planmynight\app\Controllers\Login.php(10): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 10)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\Login->index()
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-06-06 19:38:43 --> Undefined variable $user
#0 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(10): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 10)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-06-06 20:06:49 --> Array to string conversion
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(60): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 60)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(22): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 20:06:54 --> Allowed memory size of 536870912 bytes exhausted (tried to allocate 398750192 bytes)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-06-06 20:07:18 --> Array to string conversion
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(60): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 60)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(22): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 20:07:27 --> Array to string conversion
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(60): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 60)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(22): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 20:07:31 --> Allowed memory size of 536870912 bytes exhausted (tried to allocate 398750192 bytes)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-06-06 20:08:52 --> Undefined variable $places
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(60): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 60)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(27): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 20:08:56 --> Allowed memory size of 536870912 bytes exhausted (tried to allocate 398750192 bytes)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-06-06 20:09:20 --> Undefined variable $places
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(60): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 60)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(27): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 20:16:27 --> Array to string conversion
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 17)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(22): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 20:16:31 --> Allowed memory size of 536870912 bytes exhausted (tried to allocate 398750192 bytes)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-06-06 20:17:04 --> Array to string conversion
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 17)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(22): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 20:54:58 --> Undefined variable $locations
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 17)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(36): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 20:55:03 --> Allowed memory size of 536870912 bytes exhausted (tried to allocate 398750208 bytes)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-06-06 20:58:10 --> Undefined variable $locations
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 17)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(36): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 20:58:46 --> Undefined variable $locations
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 17)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(36): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 20:58:51 --> Allowed memory size of 536870912 bytes exhausted (tried to allocate 398739640 bytes)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-06-06 20:58:56 --> Undefined variable $locations
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 17)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(36): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 20:59:00 --> Allowed memory size of 536870912 bytes exhausted (tried to allocate 398740360 bytes)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-06-06 21:02:40 --> Undefined variable $locations
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 17)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(24): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 21:02:45 --> Allowed memory size of 536870912 bytes exhausted (tried to allocate 398740360 bytes)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-06-06 21:04:16 --> Undefined variable $locations
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(17): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 17)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(24): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 21:04:40 --> Undefined variable $namesurname
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(89): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 89)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(24): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-06-06 21:13:31 --> Undefined property: App\Models\Entities\Place::$getName
#0 C:\xampp\htdocs\planmynight\app\Views\myPlacesPage.php(127): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 127)
#1 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(230): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\View\View.php(232): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Common.php(1250): CodeIgniter\View\View->render('myPlacesPage', Array, true)
#4 C:\xampp\htdocs\planmynight\app\Controllers\MyPlaces.php(24): view('myPlacesPage', Array)
#5 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(931): App\Controllers\MyPlaces->index()
#6 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(439): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MyPlaces))
#7 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\CodeIgniter.php(339): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\planmynight\public\index.php(36): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\planmynight\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
